From b79cac135ca7b788892ff199231be2bfce62a4a3 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Sat, 3 May 2014 00:21:27 -0700
Subject: [PATCH:xedit] Fix man page tbl markup

'\" t is needed to work with Solaris man/nroff commands
Other changes fix issues found on all platforms

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
---
 man/xedit.man |   12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/man/xedit.man b/man/xedit.man
index 197a782..07cbf7e 100644
--- a/man/xedit.man
+++ b/man/xedit.man
@@ -1,4 +1,4 @@
-.\" t
+'\" t
 .\" $XConsortium: xedit.man,v 1.23 94/04/17 20:43:39 matt Exp $
 .\" Copyright (c) 1994  X Consortium
 .\"
@@ -95,8 +95,8 @@ Ctrl-k	Kill To End Of Line
 Meta-d	Delete Next Word
 Ctrl-l	Redraw Display
 Meta-D	Kill Word
-Ctrl-m	Newline	Meta-h
-Delete Previous Word
+Ctrl-m	Newline
+Meta-h	Delete Previous Word
 Ctrl-n	Next Line
 Meta-H	Backward Kill Word
 Ctrl-o	Newline And Backup
@@ -114,8 +114,8 @@ Ctrl-v	Next Page
 Meta-Shift Delete	Kill Previous Word
 Ctrl-w	Kill Selection	
 Meta-Backspace	Delete Previous Word
-Ctrl-y	Unkill	Meta-Shift Backspace
-Kill Previous Word
+Ctrl-y	Unkill
+Meta-Shift Backspace	Kill Previous Word
 Ctrl-z	Scroll One Line Up
 Meta-z	Scroll One Line Down
 Ctrl-_	Undo
@@ -134,7 +134,7 @@ Button 2 Down	Insert Current Selection (paste)
 Button 3 Down	Extend Current Selection
 Button 3 Motion	Adjust Selection
 Button 3 Up	End Selection (cut)
-.fi
+.TE
 .sp
 
 .SH LINE EDIT MODE
-- 
1.7.9.2

